templates/taxi/car_detail.html

```diff
@@ -6,12 +6,16 @@
     <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
       Delete
     </a>
-
     <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
       Update
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  {% if user in car.drivers.all %}
+        <a href= "{% url 'taxi:car-assign' pk=car.id %}" class="btn btn-danger link-to-page">Delete me from this car</a>
+      {% else %}
+        <a href= "{% url 'taxi:car-assign' pk=car.id %}" class="btn btn-success link-to-page">Assign me to this car</a>
```

Comment:
Ensure that the URLs for 'taxi:car-assign' are correctly defined in the `urls.py` file. If the URL pattern is incorrect, these links will not work as expected.